examples/generate-python-pydantic-models/index.ts

```diff
@@ -0,0 +1,35 @@
+import { PythonGenerator, PYTHON_PYDANTIC_PRESET } from '../../src';
+
+const generator = new PythonGenerator({
+  presets: [PYTHON_PYDANTIC_PRESET]
+});
+const jsonSchemaDraft7 = {
+  $schema: 'http://json-schema.org/draft-07/schema#',
+  type: 'object',
+  additionalProperties: false,
+  properties: {
+    optionalField: {
+      type: 'string',
+      format: 'email',
+      optional: true,
+      description: 'this field is optional'
+    },
+    // requiredField: {
+    //   type: 'string',
+    //   format: 'email',
+    //   required: true,
+    //   description: 'this field is required'
+    // },
+    noDescription: {
+      type: 'string'
+    },
+  }
```

Comment:
Damn, this is some old stuff 😆 We have to go back to [draft-02](https://datatracker.ietf.org/doc/html/draft-zyp-json-schema-02#section-5.4) for this one.

By default, all properties are optional, to define a required field you have to define it in the required field for draft-7.

I suggest we change it to the below schema 👇 

```suggestion
  $schema: 'http://json-schema.org/draft-07/schema#',
  type: 'object',
  additionalProperties: false,
  required: ["requiredField"],
  properties: {
    optionalField: {
      type: 'string',
      format: 'email',
      description: 'this field is optional'
    },
    requiredField: {
      type: 'string',
      format: 'email',
      description: 'this field is required'
    },
    noDescription: {
      type: 'string'
    },
  }
```

Reply:
Thanks for the comment, I missed that